docs/nd4j/templates/quickstart.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Quickstart
+short_title: Quick start tutorial
+description: ND4J Key features and brief samples.
+category: ND4J
+weight: 1
+---
+<!--- Comments are standard html. Tripple dash based on stackoverflow: https://stackoverflow.com/questions/4823468/comments-in-markdown -->
+
+<!--- Borrowing the layout of the Numpy quickstart to get started. -->
+
+## Introduction
+<!--- What is ND4J and why is it important. From the nd4j repo readme.  -->
+ND4J is a scientific computing library for the JVM. It is meant to be used in production environments rather than as a research tool, which means routines are designed to run fast with minimum RAM requirements. The main features are:
+* A versatile n-dimensional array object.
+* Linear algebra and signal processing functions.
+* Multiplatform functionality including GPUs.
```

Comment:
Multi-platform - or maybe a separate point - let's mention all major OSs (win/linux/osx/android), and chips (x86, arm).